components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -301,13 +304,41 @@ - (UIFont *)messageFont {
 }
 
 - (void)setBackgroundColor:(UIColor *)backgroundColor {
-  self.view.backgroundColor = backgroundColor;
+  _backgroundColor = backgroundColor;
   _tableView.backgroundColor = backgroundColor;
   _header.backgroundColor = backgroundColor;
 }
 
-- (UIColor *)backgroundColor {
```

Comment:
Why was this removed? It seems that for clients, the value of `- [controller backgroundColor]` will no longer be affected by `- [controller setBackgroundColor:]` or am I missing something?

Reply:
I added it back, if this is added then `@synthesize backgroundColor` is required.